src/qibocal/protocols/characterization/__init__.py

```diff
@@ -100,7 +102,9 @@ class Operation(Enum):
     chsh_circuits = chsh_circuits
     readout_mitigation_matrix = readout_mitigation_matrix
     twpa_frequency = twpa_frequency
+    twpa_frequency_SNR = twpa_frequency_SNR
```

Comment:
```suggestion
    twpa_frequency_SNR = twpa_frequency_snr
```
I would keep the same notation as for the other protocol